packages/manager/modules/cloud-connect/src/cloud-connect.class.js

```diff
@@ -0,0 +1,183 @@
+import { find, head, isEmpty, map, remove, set, sortBy, values } from 'lodash';
+import CloudConnectPop from './cloud-connect-pop.class';
+
+import { POP_TYPE_CONSTANT, STATUS } from './cloud-connect.constants';
+
+export default class CloudConnect {
+  constructor(cloudConnect) {
+    set(cloudConnect, 'id', cloudConnect.uuid);
```

Comment:
it's bad practice to mutate the object passed as argument to the constructor, i think what you want to do is :
```this.id = cloudConnect.uuid;```right ?

Reply:
yes